common/src/app/common/files/changes.cljc

```diff
@@ -345,7 +345,7 @@
      [:map {:title "DelComponentChange"}
       [:type [:= :del-component]]
       [:id ::sm/uuid]
-      [:main-instance {:optional true} :any]
+      [:shapes {:optional true} [:vector {:gen/max 3} :any]]
```

Comment:
We should not introduce many more any on specs, this is a big source of problems.
And this looks it expects a vector of the whole shapes, that is bit strange.